clientv3/op.go

```diff
@@ -258,6 +264,10 @@ func withTop(target SortTarget, order SortOrder) []OpOption {
 	return []OpOption{WithPrefix(), WithSort(target, order), WithLimit(1)}
 }
 
+func WithGetDeletedKVs() OpOption {
```

Comment:
It should probably match whatever it's called in DeleteResponse, which is looking very similar to RangeResponse.